virustotalv3_connector.py

```diff
@@ -740,24 +770,23 @@ def _handle_detonate_url(self, param):
         ret_val, json_resp = self._make_rest_call(self.virustotalv3_action_result,
                             URL_REPUTATION_ENDPOINT.format(id=url_id), headers=self._headers, method="get")
         if phantom.is_fail(ret_val):
-            if json_resp:
-                if json_resp['error']['code'] == 'NotFoundError' and 'Status Code: 404' in self.virustotalv3_action_result.get_message():
-                    ret_val, json_resp = self._make_rest_call(self.virustotalv3_action_result,
-                                        URL_API_ENDPOINT, body=data, headers=self._headers, method='post')
-                    if phantom.is_fail(ret_val):
-                        return ret_val
-
-                    try:
-                        scan_id = json_resp['data']['id']
-                    except KeyError:
-                        return self.virustotalv3_action_result.set_status(phantom.APP_ERROR, 'Malformed response object, missing scan_id.')
-                    return self._poll_for_result(self.virustotalv3_action_result, scan_id, self._poll_interval, wait_time)
-
             return self.virustotalv3_action_result.get_status()
 
+        if json_resp.get('error', {}).get('code') == 'NotFoundError':
```

Comment:
There is 
PASS_ERROR_CODE = {
    400: 'NotAvailableYet',
    404: 'NotFoundError',
    409: 'AlreadyExistsError'
} 
available to use. Please update similar occurrences.

Reply:
Done